vmm_tests/vmm_tests/tests/tests/x86_64.rs

```diff
@@ -40,6 +40,55 @@ async fn boot_alias_map(config: PetriVmConfig) -> anyhow::Result<()> {
     Ok(())
 }
 
+/// Basic boot test with TPM enabled.
+#[vmm_test(
+    openhcl_uefi_x64(vhd(windows_datacenter_core_2022_x64)),
+    openhcl_uefi_x64(vhd(ubuntu_2204_server_x64))
+)]
+async fn boot_with_tpm(config: PetriVmConfig) -> anyhow::Result<()> {
+    let os_flavor = config.os_flavor();
+    let config = config
+        // "OPENHCL_ENABLE_VTL2_GPA_POOL=1" is currently required to make test
+        // pass as the page pool is not enabled by default.
+        //
+        // TODO: Remove this once the page pool is always on.
+        .with_openhcl_command_line("OPENHCL_ENABLE_VTL2_GPA_POOL=1")
+        .with_tpm();
+
+    let (vm, agent) = match os_flavor {
+        OsFlavor::Windows => config.run().await?,
```

Comment:
Add a TODO comment to windows that we don't have an easy way to verify the tpm there?